internal/cmd/internal/olmv1/catalog_create.go

```diff
@@ -24,22 +35,52 @@ func NewCatalogCreateCmd(cfg *action.Configuration) *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 			i.CatalogName = args[0]
 			i.ImageSourceRef = args[1]
-
-			if err := i.Run(cmd.Context()); err != nil {
-				log.Fatalf("failed to create catalog %q: %v", i.CatalogName, err)
+			opts.Image = i.ImageSourceRef
+			if err := opts.validate(); err != nil {
+				log.Fatalf("failed to parse flags: %w", err)
+			}
+			i.DryRun = opts.DryRun
+			i.Output = opts.Output
+			i.AvailabilityMode = opts.AvailabilityMode
+			i.Priority = opts.Priority
+			i.Labels = opts.Labels
+			i.PollIntervalMinutes = opts.PollIntervalMinutes
+			catalogObj, err := i.Run(cmd.Context())
+			if err != nil {
+				log.Fatalf("failed to create catalog %q: %w", i.CatalogName, err)
```

Comment:
log.Fatal does not support wrapping of errors, we need to use `%v` here 
```suggestion
				log.Fatalf("failed to create catalog %q: %v", i.CatalogName, err)
```